testdata/dnn/tensorflow/generate_tf_models.py

```diff
@@ -66,12 +80,36 @@ def runModel(inpName, outName, name):
         writeBlob(inputData, name + '_in')
         writeBlob(outputData, name + '_out')
 
-def save(inp, out, name, quantize=False, optimize=True):
+
+def save(inp, out, group_name, name=None, quantize=False, optimize=True, is_gen_data=True):
```

Comment:
I suggest to keep origin `name`. Something like
```python
def save(inp, out, name, suffix=None, quantize=False, optimize=True):
```

Reply:
@dkurt, appropriate code corrections were done.